### PR TITLE
Integrate domains into pkg/app workflow

### PR DESCRIPTION
**Pull request by bzkrouse**
- Adds new AE widgets
- Updates to data workflow:
  - stack metadata across domains
  - example data: `labs`, `aes`
- Updates to app:
  - Nest data and settings modules in config module (each domain gets a config module)
  - Gather charts across domains into charts/reports tabs

### CODE REVIEW

File: inst/safetyGraphics_app/modules/main/main.R [hunk 1]
```diff
@@ -0,0 +1,143 @@
+# Server code for safetyGraphics App
```
Comment (jwildfire) [inline inst/safetyGraphics_app/modules/main/main.R]: Seems like some code from the config module is duplicated here? Is this intentional? If so, should we create reusable functions that can be shared between the files?

File: inst/safetyGraphics_app/modules/renderSettings/renderSettings.R [hunk 2]
```diff
@@ -45,12 +45,9 @@ source("modules/renderSettings/util/updateSettingStatus.R")
 #' \item{"settings"}{Upadted settings object based on UI/user selections}
 #' \item{"status"}{Result from validateSettings() for originally selected data + updated settings object}
 #'
-renderSettings <- function(input, output, session, data, settings, status){
+renderSettings <- function(input, output, session, data, settings, status, metadata, charts){
```
Comment (jwildfire) [inline inst/safetyGraphics_app/modules/renderSettings/renderSettings.R]: Probably worth updating the documentation here with the new inputs. Also, not entirely clear if we are using the new `charts` option in the function.

File: inst/safetyGraphics_app/global.R [hunk 3]
```diff
@@ -42,54 +42,63 @@ if (!is.null(getShinyOption("safetygraphics_charts"))){
...
+#       total_count <- preload_data_list$standard[[i]]$details[[temp_standard]]$invalid_count + valid_count
+# 
+#       fraction_cols  <- paste0(valid_count, "/" ,total_count)
+# 
+#       preload_data_list$display[[i]] <- HTML(paste0("<p>", names(preload_data_list$data)[i], " - <em style='color:green; font-size:12px;'>", "Partial ",
+#                                   standard_label, " (", fraction_cols, " data settings)",  "</em></p>"))
+#     }
+#   }
+# } else {  # otherwise use example data
+  preload_data_list <- list(labs = list(
```
[the remark this answers is not in the document]
Comment (bzkrouse) [reply inst/safetyGraphics_app/global.R]: Yes, it's commented out.  If we want to keep this functionality in this release, we can send the user-supplied data into all available domains (it'll be duplicated across domains).

File: inst/safetyGraphics_app/modules/main/main.R (hunk 1 above)
Comment (bzkrouse) [reply inst/safetyGraphics_app/modules/main/main.R]: The "main" module is leftover from my previous branch! We can get rid of it.

File: inst/safetyGraphics_app/modules/renderSettings/renderSettings.R (hunk 2 above)
Comment (bzkrouse) [reply inst/safetyGraphics_app/modules/renderSettings/renderSettings.R]: @jwildfire good catch. I can work on this.

**Pull request merged**